contrib/cirrus/packer/README.how-to-update-cirrus-vms

```diff
@@ -0,0 +1,85 @@
+This document briefly describes how to update VMs on Cirrus.
```

Comment:
I can't do that. I tense up whenever I have to read `contrib/cirrus/README.md`: it's long and covers a lot of material, but not in a way that is usually helpful to the task I wish to accomplish. I am not very smart, and I just need simple recipes: step 1, step 2, step 3. (Or, better, scripts, but this sequence is not easily scripted). It is likely that one of the next people needing to update Cirrus VMs will be me, and my mental state will be much healthier if I can find one single file listing instructions I can follow from top to bottom with no added distractions.

Reply:
Okay fair enough.  Yes, it's very difficult to document a large system like this both simply and briefly.  Some sacrifices had to be made otherwise the docs would be even larger and still nobody would read them :rofl: